.gitignore

```diff
@@ -0,0 +1,5 @@
+#Ignore MacOs default file
+.DS_Store
+
+#Ignore test.md file
+test.md
```

Comment:
- [ ] [OPTIONAL] I could see we already have test.md file uploaded to GitHub even if we add to .gitignore it will be still in there, that's because it was uploaded before being added to `.gitignore`. if you wish to get rid of it, follow [this link](https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-a-git-repository)